src/NeoSharp.Core/DI/Modules/NetworkModule.cs

```diff
@@ -28,9 +29,11 @@ public void Register(IContainerBuilder containerBuilder)
             containerBuilder.RegisterSingleton<IPeerListener, TcpPeerListener>();
             containerBuilder.RegisterSingleton<ITcpPeerFactory, TcpPeerFactory>();
 
+            containerBuilder.RegisterSingleton<IBroadcast, Server>();
```

Comment:
Could you please confirm IBroadcast and IServer are pointing to the same instance and do not create Server twice?

Reply:
Confirmed, is the same

![image](https://user-images.githubusercontent.com/3167973/41256034-b600239c-6dc8-11e8-8d44-cfa9fe9aa7ee.png)